test/bigtest/network/config.js

```diff
@@ -45,7 +45,16 @@ export default function config() {
     return schema.lines.find(request.params.id).attrs;
   });
 
-  this.get('/locations');
+  this.get('/locations', () => {
```

Comment:
@aliaksei-chumakou should we use schema here to get locations?

or it should be stubbed in test

Reply:
I don't see any reason to stub such things, it looks like regular model could be used and it could be created in test via factory